Quintor/src/main/java/com/example/quintor/ApiService.java

```diff
@@ -0,0 +1,80 @@
+package com.example.quintor;
+
+import java.io.*;
+import java.net.HttpURLConnection;
+import java.net.MalformedURLException;
+import java.net.ProtocolException;
+import java.net.URL;
+import java.nio.charset.StandardCharsets;
+
+public class ApiService {
+
+    private URL url;
+
+    public ApiService(){
+        setUrl("http://localhost:8080/MT940toJSON");
+    }
+
+    public String getRequest() throws Exception {
+        HttpURLConnection connection = setConnection("get");
+
+        BufferedReader in = new BufferedReader(new InputStreamReader(connection.getInputStream()));
+        String inputLine;
+        StringBuffer response = new StringBuffer();
+
+        while ((inputLine = in.readLine()) != null) {
+            response.append(inputLine);
+        }
+        in.close();
+
+        return response.toString();
+    }
+
+    public String postRequest(String body) throws Exception{
+        HttpURLConnection connection = setConnection("post");
+
+        OutputStream os = connection.getOutputStream();
+        byte[] input = body.getBytes(StandardCharsets.UTF_8);
+        os.write(input, 0, input.length);
+
+        BufferedReader in = new BufferedReader(new InputStreamReader(connection.getInputStream()));
+        String inputLine;
+        StringBuffer response = new StringBuffer();
+
+        while ((inputLine = in.readLine()) != null) {
+            response.append(inputLine);
+        }
+        in.close();
+
+        return response.toString();
+    }
+
+    public HttpURLConnection setConnection(String method) throws Exception {
+        HttpURLConnection connection = (HttpURLConnection) getUrl().openConnection();
+        switch (method){
+            case "get":
+                connection.setRequestMethod("GET");
+                break;
+            case "post":
```

Comment:
Is het een idee om hier een enum voor te gebruiken, zodat we het afdwingen dat ze get of post moeten gebruiken